### DISCUSSION TITLE
iOS app does not login over reverse proxy

### DISCUSSION THREAD

**Discussion by Str1atum** [immich-app/immich#8765] [Q&A]
### The bug

I setup immich with traefik via docker, port 443 and automatic SSL certificate works fine and I can access from in- and outside via https://photo.mydomain.com
I cannot, however, get the iOS app to login. I tried https://photo.mydomain.com:443/api and various short version of this URL but no success. Version is 1.101.0 on both.

### The OS that Immich Server is running on

Raspberry OS

### Version of Immich Server

1.101.0

### Version of Immich Mobile App

1.101.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

```bash
1.Login via iOS app
```

**Comment by guillaumearnx** [comment]
Same issue on Android. Last version. No app or docker logs

**Comment by guillaumearnx** [comment]
seems the app try to reach https://fqdn/api, which result in 404 error
![image](https://github.com/immich-app/immich/assets/37373941/6d776bd6-0f9a-4a70-b64e-04a2cd4f83ef)

**Comment by alextran1502** [comment]
You don't have to put port 443 into the url

**Comment by Str1atum** [comment]
> You don't have to put port 443 into the url

Doesn't work with or without the port specification

**Comment by guillaumearnx** [comment]
Personally I m not putting 443 port, only https://fqdn/

**Comment by guillaumearnx** [comment]
Oh i didnt see that WE have to put the /api. It is working now

**Comment by Str1atum** [comment]
Does not work with adding /api for me either

**Comment by alextran1502** [comment]
There might be wrong with your reverse proxy perhaps?

And make sure you are putting https not http to the url bar

**Reply by Str1atum** [reply]
Any ideas why that should not work via reverse proxy? /api should be converted seamlessly

**Comment by Str1atum** [comment]
My fault - I had a basic authelia middleware installed for my initial trials.
Should the app also work with OAuth authentification via authelia, did anyone try that?

**Reply by alextran1502** [reply]
Yes it should work, I don't have the setup for Authelia, I think you can search the discussion or ask the community on Discord ☺️

**Comment by Broyojo** [comment]
I was able to get it to work, you just have to try to login multiple times and it eventually works.
